File: 151003/Pavluchenko/publisher/src/main/java/by/bsuir/publisher/dto/responses/converters/CommentResponseConverter.java
package by.bsuir.publisher.dto.responses.converters;

import by.bsuir.publisher.domain.Comment;
import by.bsuir.publisher.dto.responses.CommentResponseDto;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring")
public interface CommentResponseConverter {
    CommentResponseDto toDto(Comment comment);
    Comment fromDto(CommentResponseDto commentResponseDto);
}

File: 151003/Yagnnish/Lab1/Dtos/ResponseDtos/ResponseConverters/CommentResponseConverter.cs
using lab_1.Domain;

namespace lab_1.Dtos.ResponseDtos.ResponseConverters
{
    public class CommentResponseConverter : BaseResponse<CommentResponseDto, Comment>
    {
        public CommentResponseDto ToDto(Comment entity)
        {
            CommentResponseDto dto = new CommentResponseDto();
            dto.content = entity.Content;
            dto.id = entity.Id;
            dto.storyId = entity.StoryId;
            return dto;
        }
    }
}
